pyproject.toml

```diff
@@ -6,6 +6,11 @@ skip_covered = true
 [tool.coverage.run]
 source = ["sshoot"]
 
+[tool.black]
+line-length = 88
+target-version = ['py37', 'py38', 'py39', 'py310', 'py311']
+include = '\.pyi?$'
+
```

Comment:
this shouldn't be needed

Reply:
Right.. I initially configured the `line-lenght` to match [flake8's](https://github.com/albertodonato/sshoot/blob/327fd92ff05f1c8a132f3a99e035b7dfba9b8b36/setup.cfg#L55) setting but the changes were a bit much